x-pack/agent/_meta/agent.docker.yml

```diff
@@ -52,15 +52,15 @@ download:
   # e.g /windows-x86.zip
   sourceURI: "https://artifacts.elastic.co/downloads/beats/"
   # path to the directory containing downloaded packages
-  target_directory: "/home/elastic/downloads"
+  target_directory: "${path.data}/beats"
```

Comment:
Perhaps still call it `downloads` and not beats as this will also download endpoint?

Reply:
+1 for download